lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -335,10 +335,16 @@ public function formatDateTime(
      */
     public function convertConfigTimeToUtc($date, $format = 'Y-m-d H:i:s')
     {
+        $zendFormat = $this->getDateTimeFormat(\IntlDateFormatter::MEDIUM);
+        $zendDate = new \Zend_Date($date, $zendFormat, $this->_localeResolver->getLocale());
```

Comment:
`Zend_Date` must not be used, there were separate efforts to get rid of it.

Please rewrite implementation using only native PHP functions.

Reply:
I tried using PHP function but all function return same error while we add date like 29/giu/2016 01:22:19. I refer Magento 1 code and try to fix it. This error is throw by PHP function.

Reply:
@patelnimesh1988 please go this way: https://github.com/magento/magento2/pull/11306#discussion_r146114650 Use something locale-aware from `intl`.

Reply:
Okay